### PR TITLE
Allow phrase grouping

### PR DESCRIPTION
**Pull request by alecgibson**
This change adds a `group` option, which will group pinyin phrases
together. This is useful if you want to generate more readable pinyin.

```
我喜欢你
wǒ xǐhuān nǐ
```

### CODE REVIEW

**Comment by alecgibson** [comment]
Let me know if this is a desirable feature, and I can try to write some tests; at the moment, I don't really understand how the tests are put together.

**Comment by hotoo** [comment]
excellent feature, add test please.

**Comment by alecgibson** [comment]
@hotoo added tests and updated the `README` files

**Comment by hotoo** [comment]
What about output result:

"我喜欢朝阳"

```json
[
  [ 'wǒ' ], 
  [ 'xǐhuān'], 
  [ 'zhāoyáng', 'cháoyáng' ],
]
```

**Comment by alecgibson** [comment]
@hotoo done

**Comment by alecgibson** [comment]
Actually @hotoo what about cases like 平安 `píng'ān`?

Maybe it's simpler to leave it as an array and leave this sort of thing to the consumer?

File: lib/index.js [hunk 1]
```diff
@@ -96,6 +100,18 @@ function segment(hans) {
...
 }
 
+function groupPhrases(phrases) {
+  if (phrases.length === 1) {
+    return phrases[0];
+  }
+
+  const grouped = phrases.reduce(function(phrase, pys) {
+    return phrase + pys[0];
+  }, "");
```
Comment (hotoo) [inline lib/index.js:110]: 这里用 phrases.join('') 就可以了。

**Pull request merged**